### DISCUSSION TITLE
Charts

### DISCUSSION THREAD

**Discussion by aris-2** [huntabyte/shadcn-svelte#1174] [Ideas]
Hello, Charts were recently added to the official site https://ui.shadcn.com/charts, are there plans to add them here in the future?

**Comment by jporre** [comment]
if someone could provide a "guide" or "how to" with instructions and one example that ports from [shadcn/ui](https://ui.shadcn.com/) to svelte we can than help to make this port fast. Charts in shadcn/ui are based in [recharts](https://github.com/recharts/recharts) using react components and D3.  Perhaps we could prepare a gpt agent to make the heavy lifting ?
